src/braket/experimental/autoqasm/program/program.py

```diff
@@ -394,6 +475,32 @@ def calibration_definition(
         finally:
             self._calibration_definitions_processing.pop()
 
+    @contextlib.contextmanager
+    def verbatim_block(self) -> None:
```

Comment:
Slightly unrelated, but should we expose a box construct and verbatim as one of the options for this box construct to make this more generic?

Reply:
Yes, I'll make that change. I'll expose `ProgramConversionContext.box()` which takes an optional pragma string. I'll move the verbatim-specific checks into the calling function `verbatim()`.